### ISSUE TITLE
compile does not accept loss strings as documented

### ISSUE THREAD

**Issue by ToonTalk** [tensorflow/tfjs#1976]
#### TensorFlow.js version
1.2.8

#### Browser version
Version 76.0.3809.132 (Official Build) (64-bit)

#### Describe the problem or feature request
`model.compile({loss: 'logLoss', optimizer: 'sgd'}); // produces error: Unknown loss logLoss`
while
`model.compile({loss: tf.losses.logLoss, optimizer: 'sgd'}); // works fine`

While documentation says
loss (string|string[]|{[outputName: string]: string}|LossOrMetricFn| LossOrMetricFn[]|{[outputName: string]: LossOrMetricFn}) 

Of the few loss functions tested only the string `'meanSquaredError'` works fine.

**Comment by caisq** [comment]
Note that `log_loss` or `logLoss` is not a built-in, string-accessible loss function in keras or tensorflow.keras either. So this primarily a doc issue. Decreasing the priority.
